internal/cli/kraft/cloud/certificate/remove/remove.go

```diff
@@ -88,7 +87,7 @@ func (opts *RemoveOptions) Pre(cmd *cobra.Command, args []string) error {
 }
 
 func (opts *RemoveOptions) Run(ctx context.Context, args []string) error {
-	auth, err := config.GetKraftCloudAuthConfig(ctx, opts.token)
+	auth, err := config.GetKraftCloudAuthConfig(ctx, opts.Token)
 	if err != nil {
 		return fmt.Errorf("could not retrieve credentials: %w", err)
 	}
```

Comment:
Can you update this code to use this convention?

https://github.com/unikraft/kraftkit/blob/b495be88eb42bc7ffe28f2a33f778dca719127ff/internal/cli/kraft/cloud/instance/create/create.go#L68-L78

I.e. expose `Auth` and `Client` (the general-purpose KC client) as non-attributes? This way we can just pass in existing values from external calls.

(please also apply to img/remove.go)